runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -1539,6 +1539,11 @@ class INTERPRETER_CLASS
 			omrthread_monitor_enter(_vm->blockedVirtualThreadsMutex);
 			_currentThread->currentContinuation->nextWaitingContinuation = _vm->blockedContinuations;
 			_vm->blockedContinuations = _currentThread->currentContinuation;
+
+			if (NULL == _currentThread->currentContinuation->objectWaitMonitor->monitor->owner) {
+				/* notify unblocker if the blocking monitor is unlocked. */
+				omrthread_monitor_notify(_vm->blockedVirtualThreadsMutex);
+			}
```

Comment:
Perhaps a local for `_currentThread->currentContinuation` (it's used three times here)?